README.rst

```diff
@@ -172,6 +172,21 @@ with old Ubuntu setups.
 If you run an Ubuntu setup with Unity and have tested Mopidy-MPRIS, please
 open an issue to share your results.
 
+KDE Plasma "Media Player"
+-----------------
+
+State:
+    Not working as of Friday 27th 2020
+Tested System: 
+    KDE Plasma V 5.20.3
+    KDE Frameworks V 5.76.0
+    QT 5.15.1
+    Kernel: 5.9.10-1-MANJARO
+    **mopidy running as service**
+
+When mpris plugin is run with default session, it fails with an X11 error. 
```

Comment:
Can you provide the exact error message?

Does it work if you run Mopidy as your own user, using the default D-Bus session?

Reply:
Sorry, just booted out of Arch into my windows install. 

The error received was the same one documented here if memory serves. 

https://github.com/mopidy/mopidy-mpris/issues/9

I did not test while running as my own user. My bet is that will work fine. 

Give me a minute to reboot and test.

Reply:
Additional detail has been added as requested. 

Let me know if you have any further questions.